awx/ui/src/components/Schedule/ScheduleList/ScheduleList.js

```diff
@@ -166,7 +166,8 @@ function ScheduleList({
         headerRow={
           <HeaderRow qsConfig={QS_CONFIG}>
             <HeaderCell sortKey="name">{t`Name`}</HeaderCell>
-            <HeaderCell sortKey="unified_job_template__polymorphic_ctype__model">{t`Type`}</HeaderCell>
+            <HeaderCell>{t`Related resource`}</HeaderCell>
```

Comment:
Lets add a sort key here
```suggestion
            <HeaderCell sortKey="unified_job_template">{t`Related resource`}</HeaderCell>
```

Reply:
Done.